src/components/custom-widget-elements/Dial.vue

```diff
@@ -286,7 +329,7 @@ const subtractDialValue = (): void => {
 }
 
 const startEditingValue = (): void => {
-  if (widgetStore.editingMode || !isInput.value) return
+  if (!isInteractive.value || !isConnected.value) return
```

Comment:
```suggestion
  if (!isInteractive.value) return
```
Connectedness is checked for as part of interactivity, so checking again here is redundant.